tests/nodeos_multiple_version_protocol_feature_test.py

```diff
@@ -170,18 +206,18 @@ def areNodesInSync(nodes:[Node]):
     libBeforePreactivation = newNodes[0].getIrreversibleBlockNum()
     newNodes[0].activatePreactivateFeature()
 
-    assert areNodesInSync(newNodes), "New nodes should be in sync"
-    assert not areNodesInSync(allNodes), "Nodes should not be in sync after preactivation"
+    assert areNodesInSync(newNodes, pauseAll=True, resumeAll=False), "New nodes should be in sync"
+    assert not areNodesInSync(allNodes, pauseAll=False, resumeAll=True), "Nodes should not be in sync after preactivation"
     for node in newNodes: assert shouldNodeContainPreactivateFeature(node), "New node should contain PREACTIVATE_FEATURE"
 
     activatedBlockNum = newNodes[0].getHeadBlockNum() # The PREACTIVATE_FEATURE should have been activated before or at this block num
     assert waitUntilBlockBecomeIrr(newNodes[0], activatedBlockNum), \
-           "1st node LIB should be able to advance past the block that contains PREACTIVATE_FEATURE"
+        "1st node LIB should be able to advance past the block that contains PREACTIVATE_FEATURE"
     assert newNodes[1].getIrreversibleBlockNum() >= activatedBlockNum and \
-           newNodes[2].getIrreversibleBlockNum() >= activatedBlockNum, \
-           "2nd and 3rd node LIB should also be able to advance past the block that contains PREACTIVATE_FEATURE"
+        newNodes[2].getIrreversibleBlockNum() >= activatedBlockNum, \
+        "2nd and 3rd node LIB should also be able to advance past the block that contains PREACTIVATE_FEATURE"
     assert oldNode.getIrreversibleBlockNum() <= libBeforePreactivation, \
-           "4th node LIB should stuck on LIB before PREACTIVATE_FEATURE is activated"
+        "4th node LIB should stuck on LIB before PREACTIVATE_FEATURE is activated"
```

Comment:
change "should stuck" to "should be stuck"

Reply:
Fixed.